nextbike/prediction/geo_predict.py

```diff
@@ -4,13 +4,23 @@
 from sklearn.neighbors import KNeighborsClassifier
 
 
-# TODO: very slow and only a first prototype
-# TODO: Add docstring
 def train_pred(p_df):
+    """Train and predict KNN Classifier for trip direction.
+
+    Train and predict if a trips end is nearer the university than the start.
+    Print the accuracy of the prdiction.
+    Args:
+        p_df (DataFrame): Dataframe of trip data
+    Returns:
+        no Return
+    """
     X = p_df[["Duration", "Dist_start", "Bike Number"]]
     y = p_df["Direction"]
     X = StandardScaler().fit_transform(X)
     X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.3, random_state=42)
```

Comment:
We should explain later / and in the report, why we chose test size 0.3 and random_state 42

Reply:
Here we split the data as well...
We split it as well in "math_split.py"

Reply:
Because of thoose two splits we do so.